source/includes/administration/_pricings.md

```diff
@@ -343,7 +345,7 @@ curl -X POST "https://cloudmc_endpoint/rest/pricings" \
         {
          "pricingMode": "FLAT_FEE",
          "lowerBound": 1000.5,
-         "upperBound": 2000.0,
```

Comment:
according to this, you can just specify upperBound: null https://cloudops.atlassian.net/wiki/spaces/gokart/pages/1901658161/Design+Pricing+Tiers+Pipeline

Reply:
it has the same effect